jennite-vcf-store/src/main/conf/plugin.properties

```diff
@@ -15,6 +15,7 @@ name=jennite-vcf-store
 title=Jennite VCF Store
 description=Stores VCF on file system and do some basic processing using bcftools.
 version=${project.version}
+type=vcf-store
```

Comment:
Is this really necessary. It is redundant with the fact that it implements the VCFStoreService